src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createReport(readData(fromFileName));
+        writeToFile(report, toFileName);
+    }
+
+    private int[] readData(String fromFileName) {
+        File fromFile = new File(fromFileName);
+        int supply = 0;
+        int buy = 0;
+        String[] lineValue;
+        String nextString;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFile))) {
+            nextString = reader.readLine();
+            while (nextString != null) {
+                lineValue = nextString.split(",");
+                if (lineValue[0].equals("supply")) {
+                    supply += Integer.parseInt(lineValue[1]);
+                } else if (lineValue[0].equals("buy")) {
+                    buy += Integer.parseInt(lineValue[1]);
```

Comment:
move magic numbers and string literals to constant

Reply:
why didn`t you request these changes earlier? why every time i check my homework, everyone finds out a few new mistakes i did from the start?